views/playlist-dashboard.twig

```diff
@@ -227,8 +227,11 @@
                     button = $("<button class=\"btn btn-block btn-warning spot-action-button\">" + $container.data("widgetName") + "</button>");
                     $container.append(button);
 
-                    let delButton = $("<a class=\"XiboFormButton added btns\" title=\"{% trans "Delete" %}\" href=\"" + $container.data("widgetDel").replace(":id", $container.data("widgetId")) + "\"><i class=\"fa fa-lg fa-trash\" aria-hidden=\"true\" style=\"padding:8px 0 8px; color:#d9534f;\"></i></a>");
-                    $delButtonDiv.append(delButton);
+                    var delTempl = Handlebars.compile($("#delete-template").html());
```

Comment:
This is a really great use of Handlebars! :+1: :star: 

One tiny addition to this, can we pass in a variable with a `trans` for Delete?